payments-core/api/payments-core.api

```diff
@@ -4460,6 +4462,12 @@ public final class com/stripe/android/model/WeChatPayNextAction : com/stripe/and
 	public fun writeToParcel (Landroid/os/Parcel;I)V
 }
 
+public final class com/stripe/android/model/parsers/PaymentIntentJsonParser : com/stripe/android/model/parsers/ModelJsonParser {
+	public fun <init> ()V
+	public fun parse (Lorg/json/JSONObject;)Lcom/stripe/android/model/PaymentIntent;
+	public synthetic fun parse (Lorg/json/JSONObject;)Lcom/stripe/android/model/StripeModel;
```

Comment:
Can any of these methods continue to be `internal`?

Reply:
hmm unfortunately not, the constructor and first `fun` is required in the `PaymentIntentFixtures.kt` to create the `PaymentIntent` object, the 2nd `synthetic fun` is from `ModelJsonParser`, which cannot be changed internal because it's defined in a interface